influxdb3_server/src/http/v1.rs

```diff
@@ -106,6 +114,67 @@ impl QueryParams {
     }
 }
 
+/// Enum representing the query format for the v1/query API.
+///
+/// The original API supports CSV, JSON, and "pretty" JSON formats.
+#[derive(Debug, Deserialize, Clone, Copy)]
+#[serde(rename_all = "snake_case")]
+pub(crate) enum QueryFormat {
+    Csv,
+    Pretty,
```

Comment:
I suggest the name `JsonPretty` for this variant, vs. `Pretty`, to not confuse it with the pretty format in the newer APIs.

Reply:
Done